Added Shiny App

harshvardhaniimi opened the pull request:
Hi rweekly team,
I have added a Shiny app for your newsletter. If there are any errors in Markdown or submission, please pardon me. This is the first time I'm trying a pull request.

Thank you,
Harshvardhan

harshvardhaniimi committed:
Added Shiny App

Hi rweekly team,
I have added a Shiny app for your newsletter. If there are any errors in Markdown or submission, please pardon me. This is the first time I'm trying a pull request.

Thank you,
Harshvardhan

jonmcalder commented:
Hi Harshvardhan!

Well done on completing your first pull request and also congrats on developing this Shiny app :-)

There's nothing wrong with this pull request as is, but can I recommend that you link to the source code for your Shiny app instead? Although we generally accept contributions of fairly wide ranging content (and news) relevant to the R community,   we like to prioritise the sharing of resources which help others to learn R, and sharing your code is one way to do that.

Is this the source code for the app here? https://github.com/harshvardhaniimi/quote-generator

You can update your pull request to make this change yourself, but if you have problems, just let us know and someone in the RWeekly team can assist you.

Kind regards,
Jon

harshvardhaniimi committed:
Changed hyperlink

Changed hyperlink from Shiny app to source code of the app

harshvardhaniimi commented:
Hi Jon,
Thanks a lot for your kind reply!

The Github repo link you suggested is correct. As you advised, I've updated the pull request to include the source code. I've also added more comments in the source code—it should help any beginner to understand better.

Do let me know if I can improve it any further.

Warm regards,
Harshvardhan

jonmcalder committed:
Update draft.md

jonmcalder commented:
Cool! I edited your PR to use a `+` instead of a `*` - what you did is correct Markdown syntax, we just seem to have a convention of using `+`.

Thanks again for your contribution :-)

harshvardhaniimi commented:
This is great! Thanks a lot.

Regards,
Harshvardhan

On Tue, Nov 24, 2020 at 3:54 PM Jon Calder <notifications@github.com> wrote:

> Cool! I edited your PR to use a + instead of a * - what you did is
> correct Markdown syntax, we just seem to have a convention of using +.
>
> Thanks again for your contribution :-)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rweekly/rweekly.org/pull/809#issuecomment-732804370>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJUZCEZVTTQY6DTMLRSNX2LSROCUXANCNFSM4T7VIRHQ>
> .
>


-- 

*Harshvardhan*
Website: https://www.harsh17.in
Mobile: +91 7033734520
